cmd/detect.go

```diff
@@ -89,6 +91,11 @@ func runDetect(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	// set follow symlinks flag
+	if detector.FollowSymlinks, err = cmd.Flags().GetBool("follow-symlinks"); err != nil {
+		log.Fatal().Err(err)
```

Comment:
```suggestion
		log.Fatal().Err(err).Msg("")
```
@RickyGrassmuck one last thing... this needs to call `.Msg("")`  otherwise the line will not be logged :/ see https://github.com/zricethezav/gitleaks/pull/985 for more details

Reply:
Never used the Zerolog package before, gonna try to remember that one in the future. Appreciate you taking care of this!